.github/workflows/unit.yml

```diff
@@ -0,0 +1,25 @@
+name: unit
```

Comment:
nit: this ends up being called "unit/unit" and "verify/verify" on the github actions on the PR...could we have them be under "test/unit" or something instead?

Reply:
same names as the registry github actions

Reply:
That's how the registry GH actions are currently named, and I didn't see any evidence we have any consistent naming of workflows/jobs here. I don't have a strong opinion either way so I'm fine with renaming both of the workflow names to be `test` unless that's problematic having two separate workflows that share the same name.

Reply:
Yes, I don't know if we have an explicit convention. Looks like our OLM github actions now are under "upstream-tests/" and "e2e-tests/" so maybe putting these under "upstream-tests/" for now makes sense. 

That being said this is now solely the upstream repository so I feel like we could drop the "upstream-" in a follow-up PR.

Reply:
@exdx There may be an existing upstream issue for this, or at least I mentioned it in removing the OCP manifests from the repository issue, but I plan on tackling some more updates to the CI pipeline once we migrate off of prow (note: but not tide), so I think it's fine to keep the current workflow/job names with the intention of trying to create a more generalized naming convention in a subsequent PR.

Reply:
+1, let's revisit this topic since there isn't an existing convention. Also agreed that explicitly calling things "upstream" doesn't make sense as we migrate the downstream bits out of operator-framework.